src/pages/audioPAge/audioPage.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import BookCover from './bookCover.jpg';
+import './style.scss';
+import audioBook from './audioBook.mp3';
+import AudioPlayer from 'react-h5-audio-player';
+import 'react-h5-audio-player/lib/styles.css';
+import { MDBContainer, MDBRow, MDBCol } from 'mdbreact';
+
+const AudioPage = () => {
+  const bookName = 'Mind without body';
+  const audio = audioBook;
+  return (
+    <MDBContainer className="mt-5   ">
+      <MDBRow>
+        <MDBCol sm="5" size="12">
+          <div className="mt-5 d-flex align-items-center justify-content-center order-last ">
+            <img src={BookCover} width="353" height="498" alt="bookCover" />
+          </div>
+        </MDBCol>
+        <MDBCol sm="7" size="12">
+          <div className="mt-5">
+            <h1 className="blue-text ">{bookName}</h1>
+            <p className="mt-3 paragraph d-none d-sm-block">
```

Comment:
text-2xl gives you font-size: 1.5rem
leading-9 gives you line-height: 2.5rem
which is exactly 150% line height so you don't need to use the style 
i don't know why you used d-none and d-sm-block i believe the paragraph should always be a block so it takes it right space only if there is another reason for it  
```suggestion
            <p className="mt-3 text-2xl leading-7 text-justify d-none d-sm-block">
```

Reply:
i will apply the classes now and  i  did the d-none to disappear the paragraph when the screen is xs

Reply:
Why do you want it to disappear?